src/config/localfile-config.h

```diff
@@ -14,6 +14,7 @@
 #define EVENTLOG     "eventlog"
 #define EVENTCHANNEL "eventchannel"
 #define DATE_MODIFIED   1
+#define MIN_EVENTCHANNEL_REC_TIME 5
```

Comment:
Changing this define to `DEFAULT_EVENTCHANNEL_REC_TIME`. The minimum allowed value will be one second.